test/lit/passes/inlining-max-comibned-size.wat

```diff
@@ -0,0 +1,735 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
```

Comment:
The name of the testcase has a typo:
```
test/lit/passes/inlining-max-comibned-size.wat
```
should be
```
test/lit/passes/inlining-max-combined-size.wat
```

Reply:
oops, fixed.